extensions/amp-inline-gallery/amp-inline-gallery.md

```diff
@@ -0,0 +1,225 @@
+      </div>
+      <div>
+        <code>&lt;script async custom-element="amp-base-carousel" src="https://cdn.ampproject.org/v0/amp-base-carousel.js">&lt;/script></code>
+      </div>
+    </td>
+  </tr>
+  <tr>
+    <td class="col-fourty"><strong><a href="https://amp.dev/documentation/guides-and-tutorials/develop/style_and_layout/control_layout">Supported Layouts</a></strong></td>
+    <td>
+      container
+    </td>
+  </tr>
+</table>
+
+## Usage
+
+The `<amp-inline-gallery>` component uses an `<amp-base-carousel>` to display slides. Note that scripts to load both extensions are required. Typical usage might look like:
+
+[example preview="inline" playground="true" imports="amp-inline-gallery,amp-base-carousel"]
+
+```html
+<amp-inline-gallery layout="container">
+  <amp-base-carousel
+    class="gallery"
+    layout="responsive"
+    width="3.6"
+    height="2"
+    snap-align="center"
+    loop="true"
+    visible-count="1.2"
+    lightbox
+  >
+    <amp-img
+      src="{{server_for_email}}/static/inline-examples/images/image1.jpg"
+      layout="responsive"
+      width="450"
+      height="300"
+    ></amp-img>
+    <amp-img
+      src="{{server_for_email}}/static/inline-examples/images/image2.jpg"
+      layout="responsive"
+      width="450"
+      height="300"
+    ></amp-img>
+    <amp-img
+      src="{{server_for_email}}/static/inline-examples/images/image3.jpg"
+      layout="responsive"
+      width="450"
+      height="300"
+    ></amp-img>
+  </amp-base-carousel>
+  <amp-inline-gallery-pagination layout="nodisplay" inset>
+  </amp-inline-gallery-pagination>
+</amp-inline-gallery>
+```
+
+[/example]
+
+The above example shows slides using an aspect ratio of 3:2, with 10% of a slide peeking on either side. Note that the carousel itself uses an aspect ratio of 3.6:2 since we show 1.2 slides at a time.
+
+### Using pagination indicators
+
+The pagination indicator will render using dots if there are eight or fewer slides contained in the carousel. When there are nine or more slides, the pagination indicator contain a text display of the current slide and the total number of slides, aligned to the right.
+
+Pagination can be specified as either inset (overlaying the carousel) or outset, underneath the carousel. If you want to use different styles for different screen sizes, you can do something like the following:
+
+```html
+<amp-inline-gallery layout="container">
+  <amp-base-carousel>…</amp-base-carousel>
+  <amp-inline-gallery-pagination
+    media="(max-width: 599px)"
+    layout="nodisplay"
+    inset
+  >
+  </amp-inline-gallery-pagination>
+  <amp-inline-gallery-pagination
+    media="(min-width: 600px)"
+    layout="fixed-height"
+    height="24"
+  >
+  </amp-inline-gallery-pagination>
+</amp-inline-gallery>
+```
+
+Where `media` is a [CSS Media Query](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries), which controls which pagination configuration to display.
+
+### Using thumbnails
+
```

Comment:
Insert a section above here:
"Display thumbnails by including the `<amp-inline-gallery-thumbnails>` element within the `<amp-inline-gallery">` component." 

Small code sample here please :)